client/src/components/utils/appHelper.js

```diff
@@ -97,10 +132,21 @@ const getGoogleMapsPlacePredictions = (query, props, interactionTypes) => {
   );
 };
 
+/**
+ * Get search bar hint text.
+ * @param  {Object} props            Props passed into the App component
+ * @param  {Object} interactionTypes Available immutable interaction types
+ * @return {String}                  Hint text
+ */
 const getSearchBarHintText = (props, interactionTypes) => (
   props.interactionType === interactionTypes.SEARCHING_ORIGIN ? 'Search for starting point' : 'Search for destination'
 );
 
+/**
+ * Open search cards.
+ * @param  {Object} props            Props passed into the App componet
+ * @param  {Object} interactionTypes Available immutable interaction types
+ */
 const openSearchCards = (props, interactionTypes) => {
```

Comment:
I think I wrote this helper function earlier on and forgot to update it--

A better name would be "changeInteractionToSearchDestination".

This should only be called when the search bar is clicked during VIEWING_MAP. **The if statement is incorrect here.** It should only check: "props.interactionType === interactionTypes.VIEWING_MAP"

_**Let me know if you would like me to make the changes instead.**_